File: bitget-golang-sdk-api/pkg/model/mix/trace/closetrackorderreq.go
package trace

/**
 * @Author: bitget-sdk-team
 * @Date: 2022-09-30 10:46
 * @DES: Dealer closing request
 */
type CloseTrackOrderReq struct {
	/**
	 * The tracking order number comes from the trackingNo of the current interface with the order
	 */
	TrackingNo string `json:"trackingNo"`
	/**
	 * Currency pair
	 */
	Symbol string `json:"symbol"`
}

File: bitget-node-sdk-api/src/lib/model/mix/trace/CloseTrackOrderReq.ts
export class CloseTrackOrderReq{
    /**
     * Currency pair
     */
    private _symbol!:string;
    /**
     * The tracking order number comes from the trackingNo of the current interface with the order
     */
    private _trackingNo!:string;


    get symbol(): string {
        return this._symbol;
    }

    set symbol(value: string) {
        this._symbol = value;
    }

    get trackingNo(): string {
        return this._trackingNo;
    }

    set trackingNo(value: string) {
        this._trackingNo = value;
    }
}